core/README.rst

```diff
@@ -1,14 +1,19 @@
 Core Helpers for Google Cloud Python Client Library
 ===================================================
 
+|pypi| |versions|
```

Comment:
@crwilcox `google-api_core` has a GA release number (1.4.1), but `google-cloud-core` doesn't (0.28.1).  Maybe @theacodes was intending to finish moving all useful code out to `google-apicore` (#4267), and thus didn't want to bump it?

Reply:
@theacodes What should we be doing about the support status of `google-cloud-core`?

Reply:
@tseaver, I spoke with @theacodes about api-core + core yesterday. Let's just leave this for now since there will be refactoring that will make this moot.